### ISSUE TITLE
TypeError: Cannot read property 'setMilliseconds' of null when attempting to render an empty line chart with time-series data

### ISSUE THREAD

**Issue by jamesdh** [plouc/nivo#1679]
**Describe/explain the bug**
While trying to upgrade from .70.1 to .72, I kept stumbling upon a "TypeError: Cannot read property 'setMilliseconds' of null" emanating from [scales/src/timeHelpers.ts:16](https://github.com/plouc/nivo/blob/master/packages/scales/src/timeHelpers.ts#L16). I tinkered around with my scale config, tried switching to `native` format (actually using a Date instead of a string), but nothing worked. Then I realized it's happening when my data is loading and the array being passed to the `data` property is initially empty. 

**To Reproduce**
Basically just take the Storybook example at https://nivo.rocks/storybook/?path=/story/line--time-scale and render it with an empty array, and it will throw the exception

**Expected behavior**
When an empty series array is defined an empty chart should still be renderable, especially if a min/max are defined for the axis. 

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome
 - Version: Latest

**Comment by jamesdh** [comment]
My best guess is this change of behavior is a result of 4a125b8
